website/docs/theme-classic.md

```diff
@@ -25,6 +25,18 @@ module.exports = {
 };
 ```
 
+With the enabled `forceDarkMode` option you could set dark mode by default. However, in this case, the user's preference will not be taken into account until they manually sets the desired mode via toggle in the navbar.
+
+```js {4} title="docusaurus.config.js"
+module.exports = {
+  // ...
+  themeConfig: {
+    forceDarkMode: true,
```

Comment:
`defaultDarkMode` would be a better name IMO. `forceDarkMode` sounds like we're forcing the site to be permanently in dark mode.